pypit/arutils.py

```diff
@@ -200,113 +205,109 @@ def dummy_fitsdict(nfile=10, spectrograph='shane_kast_blue', directory='./'):
-
-    Returns
-    -------
-
-    """
-    from pypit import arparse
-    # Dummy argflag
-    if spectrograph != 'shane_kast_blue':
-        msgs.error("Only setup for Kast Blue")  # You will need to fuss with scidx
-    argf = arparse.get_argflag_class(("ARMLSD", spectrograph))
-    argf.init_param()
-    if pypitdir is None:
-        pypitdir = __file__[0:__file__.rfind('/')]
-    # Run specific
-    argf.set_param('run pypitdir {0:s}'.format(pypitdir))
-    argf.set_param('run spectrograph {:s}'.format(spectrograph))
-    argf.set_param('run directory science ./')
-    # Dummy spect
-    spect = arparse.get_spect_class(("ARMLSD", spectrograph, "dummy"))
-    lines = spect.load_file(base=True)  # Base spectrograph settings
-    spect.set_paramlist(lines)
-    lines = spect.load_file()
-    spect.set_paramlist(lines)
-    if set_idx:
-        for jj, key in enumerate(spect._spect.keys()):
-            if key in ['det']:
-                continue
-            if 'index' in spect._spect[key].keys():
-                if spectrograph == 'shane_kast_blue':  # Science frames from idx = 5 to 9
-                    assert nfile == 10
-                for kk in [5,6,7,8,9]:
-                    if key == 'science':
-                        spect._spect[key]['index'] += [np.array([kk])]
-                    elif key == 'arc':
-                        spect._spect[key]['index'] += [np.array([1])]
-                    elif key == 'standard':
-                        spect._spect[key]['index'] += [np.array([4])]
-                    elif key == 'bias':
-                        spect._spect[key]['index'] += [np.array([0])]
-                    elif key == 'trace':
-                        spect._spect[key]['index'] += [np.array([2,3])]
-                    elif key == 'pixelflat':
-                        spect._spect[key]['index'] += [np.array([2,3])]
-    arparse.init(argf, spect)
-    return
-
-
-def dummy_specobj(fitsdict, det=1, extraction=True):
-    """ Generate dummy specobj classes
-    Parameters
-    ----------
-    fitsdict : dict
-      Expecting the fitsdict from dummy_fitsdict
-    Returns
-    -------
-
-    """
-    from astropy import units as u
-    from pypit import arspecobj
-    shape = fitsdict['naxis1'][0], fitsdict['naxis0'][0]
-    config = 'AA'
-    scidx = 5 # Could be wrong
-    xslit = (0.3,0.7) # Center of the detector
-    ypos = 0.5
-    xobjs = [0.4, 0.6]
-    specobjs = []
-    for xobj in xobjs:
-        specobj = arspecobj.SpecObjExp(shape, config, scidx, det, xslit, ypos, xobj)
-        # Dummy extraction?
-        if extraction:
-            npix = 2001
-            specobj.boxcar['wave'] = np.linspace(4000., 6000., npix)*u.AA
-            specobj.boxcar['counts'] = 50.*(specobj.boxcar['wave'].value/5000.)**-1.
-            specobj.boxcar['var']  = specobj.boxcar['counts'].copy()
-        # Append
-        specobjs.append(specobj)
-    # Return
-    return specobjs
+#def dummy_self(inum=0, fitsdict=None, nfile=10):
```

Comment:
Might as well just remove this code as it has moved.

Reply:
done